controller/src/beerocks/master/wfa_ca.cpp

```diff
@@ -43,6 +43,32 @@ static std::string check_forbidden_chars(const std::string &str)
     return forbidden_chars_found;
 }
 
+/**
+ * @brief Checks that the given string is in a hex notation
+ * 
+ * @param[in] str String to check hex notation.
+ * @return true if string is valid hex notation, else false.
+ */
+static bool validate_hex_notation(const std::string &str)
+{
+    // Each value must start with 0x
+    if (str.substr(0, 2) != "0x") {
+        return false;
```

Comment:
I believe the 0x is optional. If you look at the examples (in general, not specifically for send 1905), some of them do have a 0x and some of them don't.

We could make the argument non-const and remove the 0x here. That would simplify the rest of the code. It's a bit ugly though.

Reply:
"0x" prefix is not optional. It is mentioned on CAPI specifications that "0x" prefix is an example to hex string.
Did you see any DEV_SEND_1905 with TLVs that show otherwise?

Reply:
The "TLV Value" example has `Hex value: Ex.: {11:22:33:44:55:66 aa:bb:cc:dd:ee:f0}`.

The "Data types" section says "String using the 0-9 and A-F characters, e.g 1B234A3467"